wasmer/api.go

```diff
@@ -18,21 +18,13 @@
 package wasmer
 
 import (
-	"io/ioutil"
-	"path/filepath"
-
 	"mosn.io/proxy-wasm-go-host/proxywasm/common"
 )
 
-func NewWasmerInstanceFromFile(path string) common.WasmInstance {
-	bytes, err := ioutil.ReadFile(filepath.Clean(path))
-	if err != nil {
-		return nil
-	}
-
-	vm := NewWasmerVM()
+func NewInstanceFromBinary(wasmBytes []byte) common.WasmInstance {
```

Comment:
I will resolve any signature change in mosn when comes time to update deps.

Reply:
could keep both `NewWasmerInstanceFromFile` and `NewInstanceFromBinary`